content/_code-samples/build-a-wallet/js/5_send-xrp.js

```diff
@@ -0,0 +1,145 @@
+const { app, BrowserWindow, ipcMain } = require('electron')
+const path = require('path')
+const xrpl = require("xrpl")
+const { sendXrp } = require('./library/5_helpers')
+
+const testnetUrl = "wss://s.altnet.rippletest.net:51233"
+
+let reserveBaseXrp = null
+let reserveIncrementXrp = null
+let client = null
+let wallet = null
+
+const fixedAddress = 'rn95xwUymaMyzAKnZUGuynjZ6qk9RzV4Q7'
+const fixedSeed = 'sEdSMHvzzWCLSuh2BiTgQihfyARVAKw'
+
+const prepareAccountData = (rawAccountData) => {
+    const numOwners = rawAccountData.OwnerCount || 0
+
+    let xrpReserve = null
+    if (reserveBaseXrp && reserveIncrementXrp) {
+        //TODO: Decimal?
+        xrpReserve = reserveBaseXrp + (reserveIncrementXrp * numOwners)
+    }
+
+    return {
+        classicAddress: rawAccountData.Account,
+        xAddress: xrpl.classicAddressToXAddress(rawAccountData.Account, false, true),
+        xrpBalance: xrpl.dropsToXrp(rawAccountData.Balance),
+        xrpReserve: xrpReserve
+    }
+}
+
+const prepareLedgerData = (ledger) => {
+    reserveBaseXrp = xrpl.dropsToXrp(ledger.reserve_base)
+    reserveIncrementXrp = xrpl.dropsToXrp(ledger.reserve_inc)
+
+    return ledger
+}
+
+const prepareTxData = (transactions) => {
+    return transactions.map(transaction => ({
+        confirmed: transaction.tx.date,
+        type: transaction.tx.TransactionType,
+        from: transaction.tx.Account,
+        to: transaction.tx.Destination,
+        value: xrpl.dropsToXrp(transaction.tx.Amount),
+        hash: transaction.tx.hash
+    }))
+}
+
+const createWindow = () => {
+
+    const appWindow = new BrowserWindow({
+        width: 1024,
+        height: 768,
```

Comment:
Why is this different dimensions than before? (Not a big deal, just curious)

Reply:
It's uniform everywhere now